api/controllers/game/pass.js

```diff
@@ -56,7 +57,8 @@ module.exports = function (req, res) {
         };
         await Game.updateOne({ id: game.id }).set(gameUpdates);
         if (game.isRanked) {
-          await sails.helpers.addGameToMatch(game);
+          const currentMatch = await sails.helpers.addGameToMatch(game);
+          victory.currentMatch = currentMatch;
```

Comment:
```suggestion
          victory.currentMatch = await sails.helpers.addGameToMatch(game);
```
Unless this gets referenced below, no need to assign it to a const.